common/nistseedexpander.h

```diff
@@ -0,0 +1,37 @@
+#ifndef NISTSEEDEXPANDER_H 
+#define NISTSEEDEXPANDER_H
+
+//
+//  rng.h
+//
+//  Created by Bassham, Lawrence E (Fed) on 8/29/17.
+//  Copyright © 2017 Bassham, Lawrence E (Fed). All rights reserved.
```

Comment:
Are we allowed to redistribute this?

Reply:
We already have it in nistkatrng.c (https://github.com/PQClean/PQClean/blob/be941b28a8d568378f922d9aac7db0e2ca73d182/common/nistkatrng.c#L5)
I think it even makes sense to merge these two files so they are closer to the original rng.h distributed by NIST.

Reply:
I'm not sure I understand the problem. The `nistseedexpander.h` and `.c` file define the `seedexpander` function that was provided by NIST. HQC does not rely on `nistkatrng.h` at all, but instead just uses `randombytes.h`.